mongo/mongo.go

```diff
@@ -5,9 +5,22 @@ import (
 	"errors"
 	"time"
 
-	"github.com/globalsign/mgo"
+	mgo "github.com/gedge/mgo"
+	"github.com/gedge/mgo/bson"
 )
 
+// keep these in sync with Timestamps tags below
+const (
+	lastUpdatedKey     = "last_updated"
+	uniqueTimestampKey = "unique_timestamp"
+)
+
+// keep tags in sync with above const
+type Timestamps struct {
+	LastUpdated     time.Time           `bson:"last_updated,omitempty"     json:"last_updated,omitempty"`
```

Comment:
If you wish to omitempty on this then LastUpdated must become a pointer, as there is no `empty` value for a time.Time (only a zero value)

Reply:
agreed, updating...